_includes/header.html

```diff
@@ -1,8 +1,35 @@
 <header class="fx-row fx-row-between-xs fx-text-center">
-  <div class="fx-col fx-text-center"><a href="/{% if site.active_lang != site.default_lang %}{{ site.active_lang }}/{% endif %}">
-      <img alt="{{ site.data.general.nav.altJamulusIcon }}" width="40" height="40" src="{{ 'assets/img/jamulus-icon-2020.svg'| relative_url }}" /></a>
+  <div class="fx-col fx-text-center icon">
+    <a href="/{% if site.active_lang != site.default_lang %}{{ site.active_lang }}/{% endif %}">
+        <img alt="{{ site.data.general.nav.altJamulusIcon }}" width="40" height="40" src="{{ 'assets/img/jamulus-icon-2020.svg'| relative_url }}" />
+        <span>
+          <div>Jamulus</div>
```

Comment:
Why don't we use a heading or similar here?

Reply:
Will fix

Reply:
BTW we need to modify the language selector so that we can show it selectively on pages. Right now, it should only be on homepage, getting started and Windows/Mac/Linux install pages for EN, ES, DE, and only show on all other pages for ES.  As per this thread: https://github.com/jamulussoftware/jamulus/discussions/1302#discussioncomment-526668

Reply:
Not sure if that's practical. I'd rather add a message saying that the page you're at might be outdated and you should have a look at the english version instead.

Reply:
If readers can't know what's in an what's out of date in their translation, and if we can't remove the links, it would seem better to show a page saying "Sorry, well have paused translation of this page" or something. Even then that's fairly annoying to click on a link only to be told that. 

Perhaps better therefore to simply hard-code the links to pages that we're providing translations for?

Reply:
Hard coding would probably result in a lot more development work and it would break the whole translation. So adding a message or removing the pages which are outdated would be much cleaner.

Reply:
I don't want to bother the translators to add messages all over the place. I'll remove the affected stuff in that case.